kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -169,16 +169,39 @@
   public static final ConfigDef.Range PRODUCE_MAX_REQUESTS_PER_SECOND_VALIDATOR =
       ConfigDef.Range.between(1, Integer.MAX_VALUE);
 
+  public static final String PRODUCE_MAX_REQUESTS_GLOBAL_PER_SECOND =
+      "api.v3.produce.rate.limit.max.requests.global.per.sec";
```

Comment:
nit: I think it's better to have the `global` segment as early as possible - I'd suggest putting it just before `max`, so the config becomes `api.v3.produce.rate.limit.global.max.requests.per.sec`. The same applies for the bytes config, and it might also be applied to the constant names (although we care less about these, as they are not part of our public interface).

Reply:
As discussed, I'm going to leave this as is.